obal/data/modules/srpm.py

```diff
@@ -54,6 +56,27 @@ def copy_sources(spec_file, package_dir, sources_dir):
             run_command(["git-annex", "lock", "--force"])
 
 
+def fetch_remote_sources(source_location, source_system, sources_dir):
+    """
+    Copy RPM sources from a remote source like Jenkins to rpmbuild environment
+    """
+    if source_location and source_system:
```

Comment:
technically you'd need to check both for `is not None` only, and that's a weird API (as the params are required).

I'd personally skip that check here, and only call the method from `main()` if the vars are set *there*